### ISSUE TITLE
The colors of images rendered from ShapeNetCore dataset are not correct

### ISSUE THREAD

**Issue by le-Greg** [facebookresearch/pytorch3d#835]
## 🐛 Bugs / Unexpected behaviors
Hello,
I want to load ShapeNetCore dataset v2 using Pytorch3D, and i've made a simple code to render one example. On the image obtained, there are problems with the colors, and I don't know where it can come from.

## Instructions To Reproduce the Issue:

```
import torch

from pytorch3d.datasets import ShapeNetCore
from pytorch3d.renderer import FoVPerspectiveCameras, PointLights, RasterizationSettings, look_at_view_transform
import matplotlib.pyplot as plt

device = "cuda"
    
SHAPENET_PATH = "path to ShapeNetCore.v2 folder"
shapenet_dataset = ShapeNetCore(SHAPENET_PATH, version=2, load_textures=True)

R, T = look_at_view_transform(1.0, 1.0, 90)
cameras = FoVPerspectiveCameras(R=R, T=T, device=device)
raster_settings = RasterizationSettings(image_size=512)
lights = PointLights(location=torch.tensor([0.0, 1.0, -2.0], device=device)[None], device=device)

# 33b1973524b81c4ecafb6b9dbe847b85 model id is a moto with enough details
images_by_idxs = shapenet_dataset.render(
    model_ids=['33b1973524b81c4ecafb6b9dbe847b85'],
    device=device,
    cameras=cameras,
    raster_settings=raster_settings,
    lights=lights,
)

plt.imshow(images_by_idxs.cpu().numpy()[0, :, :, :3])
plt.show()
```

First I get these errors because there are missing 3D models in the dataset, but it's seems like it's a problem from ShapeNetCore :
```
.../pytorch3d/datasets/shapenet/shapenet_core.py:127: UserWarning: Object file not found in the model directory 3ffeec4abd78c945c7c79bdb1d5fe365 under synset directory 02958343.
  warnings.warn(msg)
.../pytorch3d/datasets/shapenet/shapenet_core.py:127: UserWarning: Object file not found in the model directory 806d740ca8fad5c1473f10e6caaeca56 under synset directory 02958343.
  warnings.warn(msg)
.../pytorch3d/datasets/shapenet/shapenet_core.py:127: UserWarning: Object file not found in the model directory f5bac2b133a979c573397a92d1662ba5 under synset directory 02958343.
  warnings.warn(msg)
.../pytorch3d/datasets/shapenet/shapenet_core.py:127: UserWarning: Object file not found in the model directory ea3f2971f1c125076c4384c3b17a86ea under synset directory 02958343.
  warnings.warn(msg)
.../pytorch3d/datasets/shapenet/shapenet_core.py:127: UserWarning: Object file not found in the model directory 7aa9619e89baaec6d9b8dfa78596b717 under synset directory 02958343.
  warnings.warn(msg)
.../pytorch3d/datasets/shapenet/shapenet_core.py:127: UserWarning: Object file not found in the model directory 5bf2d7c2167755a72a9eb0f146e94477 under synset directory 02958343.
  warnings.warn(msg)
.../pytorch3d/datasets/shapenet/shapenet_core.py:127: UserWarning: Object file not found in the model directory 5973afc979049405f63ee8a34069b7c5 under synset directory 02958343.
  warnings.warn(msg)
.../pytorch3d/datasets/shapenet/shapenet_core.py:127: UserWarning: Object file not found in the model directory 9fb1d03b22ecac5835da01f298003d56 under synset directory 02958343.
  warnings.warn(msg)
.../pytorch3d/datasets/shapenet/shapenet_core.py:127: UserWarning: Object file not found in the model directory 207e69af994efa9330714334794526d4 under synset directory 02958343.
  warnings.warn(msg)
.../pytorch3d/datasets/shapenet/shapenet_core.py:127: UserWarning: Object file not found in the model directory 2307b51ca7e4a03d30714334794526d4 under synset directory 02958343.
  warnings.warn(msg)
.../pytorch3d/datasets/shapenet/shapenet_core.py:127: UserWarning: Object file not found in the model directory 3c33f9f8edc558ce77aa0b62eed1492 under synset directory 02958343.
  warnings.warn(msg)
.../pytorch3d/datasets/shapenet/shapenet_core.py:127: UserWarning: Object file not found in the model directory 986ed07c18a2e5592a9eb0f146e94477 under synset directory 02958343.
  warnings.warn(msg)
.../pytorch3d/datasets/shapenet/shapenet_core.py:127: UserWarning: Object file not found in the model directory d6ee8e0a0b392f98eb96598da750ef34 under synset directory 02958343.
  warnings.warn(msg)
.../pytorch3d/datasets/shapenet/shapenet_core.py:127: UserWarning: Object file not found in the model directory 93ce8e230939dfc230714334794526d4 under synset directory 02958343.
  warnings.warn(msg)
.../pytorch3d/datasets/shapenet/shapenet_core.py:127: UserWarning: Object file not found in the model directory e6c22be1a39c9b62fb403c87929e1167 under synset directory 02958343.
  warnings.warn(msg)
.../pytorch3d/datasets/shapenet/shapenet_core.py:127: UserWarning: Object file not found in the model directory 4ddef66f32e1902d3448fdcb67fe08ff under synset directory 02958343.
  warnings.warn(msg)
.../pytorch3d/datasets/shapenet/shapenet_core.py:127: UserWarning: Object file not found in the model directory 407f2811c0fe4e9361c6c61410fc904b under synset directory 02958343.
  warnings.warn(msg)
.../pytorch3d/datasets/shapenet/shapenet_core.py:127: UserWarning: Object file not found in the model directory 8070747805908ae62a9eb0f146e94477 under synset directory 02958343.
  warnings.warn(msg)
.../pytorch3d/datasets/shapenet/shapenet_core.py:127: UserWarning: Object file not found in the model directory 302612708e86efea62d2c237bfbc22ca under synset directory 02958343.
  warnings.warn(msg)
```

Then :
```
.../pytorch3d/io/mtl_io.py:101: UserWarning: Texture UV coordinates outside the range [0, 1].                 The integer part will be ignored to form a repeating pattern.
  warnings.warn(msg)
.../pytorch3d/transforms/transform3d.py:782: UserWarning: R is not a valid rotation matrix
  warnings.warn(msg)
```

And the final image, it seems like there is some noise, or some faces don't have the proper colors
![image](https://user-images.githubusercontent.com/68744272/132388088-f3b1fc33-d9a2-4000-836f-dc9e16e1529c.png)

I used Conda environment, python 3.8.11, pytorch 1.8.1, cuda 11.1, pytorch3d nightly 0.5

**Comment by le-Greg** [comment]
I found this  topic #313 that had the same issue.

Incorrect colors are caused by duplicated face that overlap each other, and the inner face is not colored. Set cull_backface to True solve the problem in the RasterizationSettings. You also need to turn off the lightning to remove visual glitches

```
lights = PointLights(
    device=device, 
    location=[[0.0, 5.0, -10.0]], 
    diffuse_color=((0, 0, 0),),
    specular_color=((0, 0, 0),),
)
```

With this fix :
![image](https://user-images.githubusercontent.com/68744272/132582403-e15be265-ef88-410f-9da4-22671dc5a899.png)

Should this known issue also be mentioned in this [tutorial](https://pytorch3d.org/tutorials/dataloaders_ShapeNetCore_R2N2) ? This is where I started, and that would have been useful information to me
